packages/aws-cdk-api/lib/api.ts

```diff
@@ -0,0 +1,226 @@
+import * as core from '@aws-cdk/core';
+import { exec as runCli } from 'aws-cdk/lib';
```

Comment:
I trust that this works in the working copy... I don't fully trust that this works after publishing to NPM, based on trickery and #$%*ery we do to the CLI build before we publish.

Does it?

Reply:
Yes it does work. This minimal example can be run in a shell:
```sh
npm install aws-cdk 
node -e "const { cli } = require('aws-cdk/lib'); cli()"
```

`cli` is the existing export. I have no reason to believe `exec` would behave any different. However it's tricky to proof with the current build system.

Maybe doing the code swap first would be better?